v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-tcn/auto-ml-forecasting-demand-tcn.ipynb

```diff
@@ -0,0 +1,1470 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<!-- Both training and inferencing is done using pipelines.\n",
+    "\n",
+    "Relies on the following scripts:\n",
+    "\n",
+    "- `helper_scripts.py`: Inference script that is executed on the remote compute\n",
+    "- `register_model.py`: needed for the training stage. Registers model in the workspace.\n",
+    "- `inference_script_naive.py`: Inference script for the Naive model\n",
+    "- `inference_script_tcn.py`: Inference script for the TCN model\n",
+    "-->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import logging\n",
+    "import os\n",
+    "\n",
+    "from matplotlib import pyplot as plt\n",
+    "import pandas as pd\n",
+    "\n",
+    "import azureml.core\n",
+    "from azureml.core.experiment import Experiment\n",
+    "from azureml.core.workspace import Workspace\n",
+    "from azureml.train.automl import AutoMLConfig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This sample notebook may use features that are not available in previous versions of the Azure ML SDK."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"This notebook was created using version 1.47.0 of the Azure ML SDK\")\n",
+    "print(\"You are currently using version\", azureml.core.VERSION, \"of the Azure ML SDK\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Accessing the Azure ML workspace requires authentication with Azure.\n",
+    "\n",
+    "The default authentication is an interactive authentication using the default tenant. Executing the ws = Workspace.from_config() line in the cell below will prompt for authentication the first time it is run.\n",
+    "\n",
+    "If you have multiple Azure tenants, you can specify the tenant by replacing the ws = Workspace.from_config() line in the cell below with the following:\n",
+    "```\n",
+    "from azureml.core.authentication import InteractiveLoginAuthentication\n",
+    "auth = InteractiveLoginAuthentication(tenant_id = 'mytenantid')\n",
+    "ws = Workspace.from_config(auth = auth)\n",
+    "```\n",
+    "If you need to run in an environment where interactive login is not possible, you can use Service Principal authentication by replacing the ws = Workspace.from_config() line in the cell below with the following:\n",
+    "```\n",
+    "from azureml.core.authentication import ServicePrincipalAuthentication\n",
+    "auth = ServicePrincipalAuthentication('mytenantid', 'myappid', 'mypassword')\n",
+    "ws = Workspace.from_config(auth = auth)\n",
+    "```\n",
+    "For more details, see [this link](aka.ms/aml-notebook-auth)."
```

Comment:
Doesn't seem that customer outside Microsoft could directly use aka.ms link? Landed on a 404 page when clicking it directly. But, if I copy the link in the browser, it gives me the link that should be accessible by customers: https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/manage-azureml-service/authentication-in-azureml/authentication-in-azureml.ipynb

Reply:
Fixed.